src/GeositeFramework/js/Export.js

```diff
@@ -1,12 +1,121 @@
+/*global Backbone, _, $ */
+
 require(['use!Geosite',
+         'esri/map',
+         'esri/geometry/Extent',
+         'esri/layers/ArcGISDynamicMapServiceLayer',
+         'esri/layers/ArcGISTiledMapServiceLayer',
+         'dojo/dom-style',
          'dojo/Deferred',
          'dojo/request',
          'framework/Logger'],
-    function(N,
-             Deferred,
-             request,
-             Logger) {
+    function(N) {
     "use strict";
+        function render(view) {
+            var html = N.app.templates['template-export-window']();
+            view.$el.empty().append(html);
+            if (view.$uiContainer) {
+                view.$uiContainer.show();
+            }
+
+            return view;
+        }
+
+        function createUiContainer(view, paneNumber) {
+            var model = view.model,
+                $uiContainer = $($.trim(N.app.templates['template-export-window']({id: 'export-print-preview'})));
+
+            view.$uiContainer = $uiContainer;
+
+            var pluginCssPath = 'css/app-print.css',
+                printCssClass = 'plugin-print-css',
+                oppositePaneHideCssPath = 'css/print-hide-map' +
+                    (paneNumber === 0 ? 1 : 0) + '.css';
+
+
+            // Add the plugin css
+            addCss(pluginCssPath, printCssClass);
+
+            // Hide the possible same plugin from the opposite map pane
+            addCss(oppositePaneHideCssPath, printCssClass);
```

Comment:
This is now unnecessary since there is only one map pane. The previous version of the app allowed of split-screen map mode, which this line is protecting against.